migration-guides/v5.0.0.md

```diff
@@ -0,0 +1,21 @@
+# v5.0.0 Migration Guide
+
+_Note: this guide assumes you are upgrading from decanter v4 to v5._
+
+This version contains major updates including the upgrade to Ruby version 3.3.0. Ensure your environment is compatible with this Ruby version before proceeding.
+
+## Major Changes
```

Comment:
Does this also need to include Rails 7? Or am I misunderstanding the consequences of the actionpack upgrade?